ambry-api/src/main/java/com.github.ambry/account/Account.java

```diff
@@ -168,32 +171,27 @@ private Account(JSONObject metadata) throws JSONException {
   }
 
   /**
-   * Gets the id of the account.
-   * @return The id of the account.
-   */
-  public short getId() {
-    return id;
-  }
-
-  /**
-   * Gets the name of the account.
-   * @return The name of the account.
+   * Gets the metadata of the account in {@link JSONObject}. This will increment the snapshot version by one.
+   * @return The metadata of the account in {@link JSONObject}.
+   * @throws JSONException If fails to compose the metadata in {@link JSONObject}.
    */
-  public String getName() {
-    return name;
+  JSONObject toJson() throws JSONException {
+    return toJson(true);
   }
 
   /**
-   * Gets the metadata of the account in {@link JSONObject}.
+   * Gets the metadata of the account in {@link JSONObject}. This will increment the snapshot version by one.
+   * @param incrementSnapshotVersion {@code true} to increment the snapshot version by one.
    * @return The metadata of the account in {@link JSONObject}.
    * @throws JSONException If fails to compose the metadata in {@link JSONObject}.
    */
-  JSONObject toJson() throws JSONException {
+  JSONObject toJson(boolean incrementSnapshotVersion) throws JSONException {
```

Comment:
any reason to make this a public function and get rid of the other one? the other one is a little bit unintuitive IMO